### ISSUE TITLE
pip install fails with `use_2to3 is invalid`

### ISSUE THREAD

**Issue by cowlinator** [sunshowers-code/py-ntfs#14]
Command: `python -m pip install ntfsutils`

Output:
```
Collecting ntfsutils
  Downloading ntfsutils-0.1.4.tar.gz (7.3 kB)
  Preparing metadata (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: 'D:\temp\zzz\venv\Scripts\python.exe' -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\cowlinator\\AppData\\Local\\Temp\\1\\pip-install-ai61tbm4\\ntfsutils_6a935ee7350943a6b886e7e7fd8f8b9b\\setup.py'"'"'; __file__='"'"'C:\\Users\\cowlinator\\AppData\\Local\\Temp\\1\\pip-install-ai61tbm4\\ntfsutils_6a935ee7350943a6b886e7e7fd8f8b9b\\setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base 'C:\Users\cowlinator\AppData\Local\Temp\1\pip-pip-egg-info-n7h_d0q6'
       cwd: C:\Users\cowlinator\AppData\Local\Temp\1\pip-install-ai61tbm4\ntfsutils_6a935ee7350943a6b886e7e7fd8f8b9b\
  Complete output (1 lines):
  error in ntfsutils setup command: use_2to3 is invalid.
```

Environment:
Windows 10 20H2 (OS Build 19042.1348)
Python 3.7.4
pip 21.3.1
wheel 0.37.0
setuptools 59.6.0

No other python modules (`pip freeze` returns nothing)

**Comment by sunshowers** [comment]
Fixed by #13 -- release coming out shortly.

**Comment by cowlinator** [comment]
Thank you!
